source/index.ts

```diff
@@ -0,0 +1,19 @@
+interface IHash {
+	[key: string]: unknown;
+}
+
+const lowercaseKeys = (input: object): IHash => {
+	const ret = {};
+
+	Object.keys(input).forEach((key: string) => {
```

Comment:
had to switch from `Object.entries` to `Object.keys`, to support node 6 (because package.json says so).